packages/shared/src/components/streak/popup/DayStreak.tsx

```diff
@@ -50,9 +55,17 @@ export function DayStreak({
         className={classNames(
           className,
           iconSizeToClassName[size],
-          streak === Streak.Freeze && 'bg-text-disabled',
+          streak === Streak.Freeze &&
+            'flex cursor-pointer items-center justify-center bg-text-disabled text-transparent laptop:hover:text-surface-secondary',
         )}
-      />
+        onClick={() => {
+          if (streak === Streak.Freeze && onClick && isLaptop) {
```

Comment:
This should only work on laptop?
See on tablet/mobile we don't show the hover/action.
(mobile is probably fine, but not sure on the tablet size?)
![Screenshot 2024-08-02 at 11 31 05](https://github.com/user-attachments/assets/fc4a882d-d482-4f0a-9941-56a3c7de795a)
![Screenshot 2024-08-02 at 11 30 54](https://github.com/user-attachments/assets/f3b75644-d9fe-476e-99c9-9113ae1584a2)

Reply:
From design
> on mobile will not be available to press

Tbh, we could probably still allow the click on mobile, we just wouldn't be able to see the edit icon on hover, as touch does not have hover

Reply:
Yeah would probably be easier to support smaller screens?

Reply:
👍 will make it possible to click the weekend dots on all devices. And check that we don't render the tooltip on mobile